Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1072,8 +1072,8 @@ u8 NetPlayClient::LocalWiimoteToInGameWiimote(u8 local_pad)
 	// The logic we have here is that the local slots always
 	// go in order.
 	int local_pad_count = -1;
-	int ingame_pad = 0;
-	for (; ingame_pad < 4; ingame_pad++)
+	unsigned char ingame_pad = 0;
+	for (; ingame_pad < 4; ++ingame_pad)
 	{
```

Comment:
All of these and above can be left as ints (or made unsigned ints). There's no need to use a narrower data type.